specimen/contexts/archive-page.html

```diff
@@ -294,7 +297,32 @@ <h2>Support Our Work</h2>
 
     </footer>
 
-<script src="../../../src/js/vocabulary.js"></script>
+    <div class="scroll-container">
+        <a href="#top" class="scroll-to-top icon-attach fa-angle-up" aria-label="Scroll to top"></a>
+    </div>
+
+<!--<script src="../../../src/js/vocabulary.js"></script>-->
+    <script>
+        document.addEventListener('DOMContentLoaded', function () {
```

Comment:
Any JS would need to be added to a separate .js file and not embedded within the HTML structure.